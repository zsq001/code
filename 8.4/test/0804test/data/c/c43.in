100
7 2 8 4 7 5 4 5 10 4 3 4 6 2 9 2 5 8 3 7 9 4 4 2 1 2 7 8 1 8 8 9 8 8 6 9 1 5 5 2 6 6 2 1 8 8 9 9 5 5 7 7 9 9 8 10 9 5 2 6 10 1 4 4 4 1 3 8 3 3 10 10 5 8 8 6 3 1 1 9 10 6 4 5 4 6 4 3 9 3 4 7 4 5 8 4 9 4 9 6
7
5
2
3
9
6
10
9
