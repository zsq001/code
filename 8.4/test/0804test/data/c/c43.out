7
25
12
2
6
2
2
